content/docs/command-reference/commit.md

```diff
@@ -19,10 +19,10 @@ positional arguments:
 
 The `dvc commit` command is useful for several scenarios, when data already
 tracked by DVC changes: when a [stage](/doc/command-reference/run) or
-[pipeline](/doc/command-reference/dag) is in development/experimentation; when
-manually editing or generating DVC <abbr>outputs</abbr>; or to force update the
-`dvc.lock` or `.dvc` files without reproducing stages or pipelines. These
-scenarios are further detailed below.
+[pipeline](/doc/command-reference/dag) is in development/experimentation; to
+force-update the `dvc.lock` or `.dvc` files without reproducing stages or
+pipelines; or when DVC <abbr>outputs</abbr> are added to a stage, manually
+generated, or edited. These scenarios are further detailed below.
```

Comment:
> when DVC <abbr>outputs</abbr> are added to a stage, manually generated, or edited

I don't even know what we were trying to say here, actually :/ Please go over this phrasing again, it should be simple language that makes sense, not a riddle that readers will just shrug off/ignore/skip. Just a general statement that doesn't have all the details is better here, since it will be explained in detail below in the bullets. No need to force a complete explanation in a single sentence, try to make it general yet correct.

Reply:
Have updated this. Tried to make it simpler and more general.